app/Console/Commands/Members/ImportMembers.php

```diff
@@ -3,15 +3,15 @@
 namespace App\Console\Commands\Members;
 
 use App\Console\Commands\Command;
-use App\Libraries\AutoTools;
```

Comment:
Can this file be deleted?

Reply:
It will be able to but I want to keep it in just for now in case we come to migrate anything like prat and want to reference it in the future.